### DISCUSSION TITLE
How to use this after https://epiforecasts.io shuttered

### DISCUSSION THREAD

**Discussion by cyberphotocopy** [epiforecasts/covid-rt-estimates#212] [Q&A] [answered]
There is no method given on how to use these files and view the data because all it says is this, "We have since restarted our analysis pipeline to produce estimates at the national level once a week, published on the [GitHub repository](https://github.com/epiforecasts/covid-rt-estimates) but no longer featured on the EpiForecasts website."  https://epiforecasts.io/posts/2022-03-25-rt-reflections/

**Comment by sbfnk** [comment, marked as answer]
These are no longer produced. The last estimates produced (3 months ago) can be accessed here: https://github.com/epiforecasts/covid-rt-estimates/tree/master/national/cases/summary
